alot/defaults/alot.rc.spec

```diff
@@ -248,6 +248,9 @@ reply_account_header_priority = force_list(default=list(From,To,Cc,Envelope-To,X
 #           history file might get *very* long.
 history_size = integer(default=50)
 
+# The number of seconds to wait between calls to the loop_hook
+loop_period = integer(default=300)
```

Comment:
what about "loop_hook_frequency" or "loop_hook_period"?

Yes, `LoopingCall`'s interval argument is defined in seconds.